test_materialized_sql_view/tests/test_pg_materialized_view_manager.py

```diff
@@ -0,0 +1,71 @@
+from openerp.tests.common import SingleTransactionCase
+from openerp.addons.materialized_sql_view.model.abstract_materialized_sql_view import PGMaterializedViewManager  # noqa
+from openerp.addons.materialized_sql_view.model.abstract_materialized_sql_view import PGNoMaterializedViewSupport  # noqa
+from openerp.addons.materialized_sql_view.model.abstract_materialized_sql_view import PG090300  # noqa
```

Comment:
This is a sign that your naming is too long. Consider shortening them.

Otherwise, this works:
```python
from openerp.addons.materialized_sql_view.model.abstract_materialized_sql_view\
    import (
        PGMaterializedViewManager,
        PGNoMaterializedViewSupport,
        PG090300,
    )
```